src/features/Authentication/organisms/SignUpForm/SignUpForm.js

```diff
@@ -0,0 +1,21 @@
+import React, { Fragment, Component } from 'react';
+// import { useNavigate } from 'react-router-dom';
+// import Button from "@mui/material/Button";
+//
+// import CREATE_USER_MUTATION from 'mutations/CreateUser/createUser';
+// import { useMutation } from 'urql';
```

Comment:
`// import CREATE_USER_MUTATION from 'mutations/CreateUser/createUser';
// import { useMutation } from 'urql';`

this will be used later, commented for now